Makefile

```diff
@@ -142,10 +142,10 @@ gcloud-auth:
 # docker images
 # =============================================================================
 image:
-	DOCKER_BUILDKIT=1 docker build -t $(DOCKER_TAG_DOCKERHUB) .
+	docker buildx build --platform linux/amd64,linux/arm64 -t $(DOCKER_TAG_DOCKERHUB) .
 
-imagepush: image
-	docker push $(DOCKER_TAG_DOCKERHUB)
+imagepush:
+	docker buildx build --push --platform linux/amd64,linux/arm64 -t $(DOCKER_TAG_DOCKERHUB) .
```

Comment:
i think this needs to be re-jiggered based on how buildx works and how it needs a `builder` defined for it to use:

this has worked for me although the first step of ensuring any orphaned buildx builders could be improved be parsing the output of `docker buildx ls`

since buildx needs some setup, i do this on my end for multi-arch's:

ensure i cleared out the buildx builder in case it got interrupted before
```
docker buildx rm <builder-name>
```
then:
```
docker buildx create --name <builder-name>
docker buildx use <builder-name>
docker buildx build --tag <remote_tag> . --platform linux/amd64,linux/arm64,linux/arm/v7 --push
```
then clear out the buildx builder so it's ready for the next time i run it
```
docker buildx rm <builder-name>
```

Reply:
i got it running in `sso` like this: https://github.com/buzzfeed/sso/pull/319